src/vs/platform/files/test/electron-browser/normalizer.test.ts

```diff
@@ -9,10 +9,9 @@ import { FileChangeType, FileChangesEvent } from 'vs/platform/files/common/files
 import { URI as uri } from 'vs/base/common/uri';
 import { IDiskFileChange, normalizeFileChanges, toFileChanges } from 'vs/platform/files/node/watcher/watcher';
 import { Event, Emitter } from 'vs/base/common/event';
-import { ExtUri } from 'vs/base/common/resources';
 
 function toFileChangesEvent(changes: IDiskFileChange[]): FileChangesEvent {
-	return new FileChangesEvent(toFileChanges(changes), new ExtUri(() => !platform.isLinux));
+	return new FileChangesEvent(toFileChanges(changes), !platform.isLinux);
```

Comment:
The `!platform.isLinux` is slightly fishy... I know it's caused by the TST and I don't have a much better idea. The challenge is that it cannot really work with the IExtUri interface...

Reply:
@jrieken this is only from a test right, the actual (and only) usage is proper here:

https://github.com/microsoft/vscode/blob/2dcc4560a0d2cd56c5555d30cddc55347b3f97af/src/vs/platform/files/common/fileService.ts#L58-L58